integrationTests/IntegrationTest.php

```diff
@@ -298,9 +298,9 @@ public function addFunding(
 
         $db->executeStatement(<<<SQL
             INSERT INTO Fund (name, salesforceId, salesforceLastPull, createdAt, updatedAt, fundType,
-                              currencyCode) VALUES 
+                              currencyCode, allocationOrder) VALUES 
                 ('Some test fund', '$fundSfID', '$nyd', '$nyd', '$nyd', '{$fundType->value}',
-                 'GBP')
+                 'GBP', 1)
```

Comment:
Probably worth having this a bit more realistic with 
```suggestion
                 'GBP', {$fundType->getAllocationOrder()})
```